performance-metrics/src/performance_tests.rs

```diff
@@ -666,12 +688,33 @@ pub fn performance_block_io(control: &PerformanceTestControl) -> f64 {
             .expect("Failed to hotplug test disk image")
             .success());
 
+        #[cfg(target_arch = "aarch64")]
+        scp_to_guest(
+            Path::new(fio_path),
+            Path::new(fio_remote_path),
+            &guest.network.guest_ip,
+            //DEFAULT_SSH_RETRIES,
+            1,
+            DEFAULT_SSH_TIMEOUT,
+        )
+        .map_err(Error::Scp)
+        .unwrap();
+
+        #[cfg(target_arch = "x86_64")]
         let fio_command = format!(
             "sudo fio --filename=/dev/vdc --name=test --output-format=json \
             --direct=1 --bs=4k --ioengine=io_uring --iodepth=64 \
             --rw={} --runtime={} --numjobs={}",
             fio_ops, test_timeout, num_queues
         );
+
+        #[cfg(target_arch = "aarch64")]
+        let fio_command = format!(
+            "sudo /tmp/fio --filename=/dev/vdc --name=test --output-format=json \
```

Comment:
...so that here we can use the same cmdline as x86

Reply:
Yes please do so, otherwise the test is a bit confusing to read.

Reply:
I tried to add another commit https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3838/commits/7c98e5eae8aa8d3b72970bdc6656c677e22d8859 to solve this issue.